webclient/src/types/server.tsx

```diff
@@ -54,6 +54,13 @@ export class Host {
 }
 
 export const DefaultHosts: Host[] = [
+  {
+    name: 'Chickatrice',
+    host: 'mtg.chickatrice.net',
+    port: '443',
+    localPort: '4748',
```

Comment:
Is this something that should be documented in the README file?

Reply:
Possibly, the README is missing a lot though.